src/util/constraints/clientRegistration.ts

```diff
@@ -14,14 +14,4 @@ export default {
 			allowEmpty: false,
 		},
 	},
-	firstName: {
-		presence: {
-			allowEmpty: false,
-		},
-	},
-	lastName: {
-		presence: {
-			allowEmpty: false,
-		},
-	},
```

Comment:
similar to my comment on `register.ts` first and last name weren't present in the client registration form or the clients db table so I dropped these constraints in order to be able test a client registration.